source/api_documentation/network_integration/advertisers/_post_advertiser.rst

```diff
@@ -6,10 +6,19 @@
 
   Create an advertiser with users
 
-  Endpoint:
+  Endpoints:
 
   ``https://invoca.net/api/@@NETWORK_API_VERSION/<network_id>/advertisers.json``
 
+  or...
+
+  ``https://invoca.net/api/@@NETWORK_API_VERSION/<network_id>/advertisers/<advertiser_id_from_network>.json``
+
+  If you prefer the ``/advertisers.json`` endpoint but receive an error, your network will need to be enabled to auto-generate ids from network.
```

Comment:
I reworded these to read more like instructions than questions. Note the BLA BLA part: I'm not sure what the contact info is. Please find similar wording elsewhere in our docs and copy/paste.

```
Use the ``/advertisers.json`` endpoint if you want to specify the ID in the post body, or allow the ID to be auto generated. (Your account can be configured to automatically generate IDs in a format you specify. Contact BLA BLA for more information)

Use the ``/advertisers/<advertiser_id_from_network>.json`` endpoint if you want to specify the ID in the URL. Note: If you specify the ID in the URL and the body, the body's id will take precedence over the URL.
```